### PR TITLE
Topic card view group list

### PR DESCRIPTION
**Pull request by CHenrie**

### CODE REVIEW

File: src/GroupedList.js [hunk 1]
```diff
@@ -169,33 +169,41 @@ define('Sage/Platform/Mobile/GroupedList', [
             }
             else if (feed['$resources'])
             {
-                var o = [];
-
+                //var o = [];
```
Comment (jbest84) [inline src/GroupedList.js]: Remove commented out code

File: src/GroupedList.js [hunk 2]
```diff
@@ -169,33 +169,41 @@ define('Sage/Platform/Mobile/GroupedList', [
...
+                var docfrag = document.createDocumentFragment();
                 for (var i = 0; i < feed['$resources'].length; i++)
```
Comment (jbest84) [inline src/GroupedList.js:173]: Multiple vars

File: src/GroupedList.js [hunk 3]
```diff
@@ -169,33 +169,41 @@ define('Sage/Platform/Mobile/GroupedList', [
...
                 {
                     var entry = feed['$resources'][i],
                         entryGroup = this.getGroupForEntry(entry);
-
+                    var rowNode;
```
Comment (jbest84) [inline src/GroupedList.js:177]: Multiple vars

File: src/GroupedList.js [hunk 4]
```diff
@@ -169,33 +169,41 @@ define('Sage/Platform/Mobile/GroupedList', [
...
                     if (entryGroup.tag != this._currentGroup)
                     {
-                        if (o.length > 0)
-                            domConstruct.place(o.join(''), this._currentGroupNode, 'last');
-
-                        o = [];
-
+                        //if (o.length > 0)
```
Comment (jbest84) [inline src/GroupedList.js]: Remove commented out code

File: src/GroupedList.js [hunk 5]
```diff
@@ -169,33 +169,41 @@ define('Sage/Platform/Mobile/GroupedList', [
...
                     }
 
                     this.entries[entry.$key] = entry;
-
-                    o.push(this.rowTemplate.apply(entry, this));
-
+                    rowNode = domConstruct.toDom(this.rowTemplate.apply(entry, this));
+                    this.onApplyRowTemplate(entry, rowNode);
+                    docfrag.appendChild(rowNode);
+                    //o.push(this.rowTemplate.apply(entry, this));
```
Comment (jbest84) [inline src/GroupedList.js]: Remove commented out code

**Comment by CHenrie** [comment]
@jbest84  Please Review

File: src/GroupedList.js [hunk 6]
```diff
@@ -169,33 +169,41 @@ define('Sage/Platform/Mobile/GroupedList', [
...
+                    //this.onApplyRowTemplate(entry, rowNode);
                 }
 
-                if (o.length > 0)
-                    domConstruct.place(o.join(''), this._currentGroupNode, 'last');
+               // if (o.length > 0)
+                //     domConstruct.place(o.join(''), this._currentGroupNode, 'last');
```
Comment (jbest84) [inline src/GroupedList.js]: Remove commented out code

File: src/List.js [hunk 7]
```diff
@@ -1295,21 +1300,22 @@ define('Sage/Platform/Mobile/List', [
             }
             else if (feed['$resources'])
             {
-                var o = [];
-
+                var docfrag = document.createDocumentFragment();
                 for (var i = 0; i < feed['$resources'].length; i++)
```
Comment (jbest84) [inline src/List.js:1304]: Multiple vars

**Comment by jbest84** [comment]
@CHenrie Accept

**Pull request merged**